ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2135,5 +2135,81 @@
         "name": "type-java.svg",
         "mediaType": "image/svg+xml"
     }
+  },
+  {
+    "id": "spring-boot",
+    "creator": "Gytis Trikleris",
+    "name": "Spring Boot",
+    "description": "Spring Boot Stack on CentOS.",
+    "scope": "advanced",
+    "tags": [
+      "Java",
+      "JDK",
+      "Maven",
+      "Spring Boot",
+      "CentOS",
+      "Git"
+    ],
+    "components": [
+      {
+        "name": "CentOS",
+        "version": "---"
+      },
+      {
+        "name": "JDK",
+        "version": "1.8.0_45"
```

Comment:
I've used the same version as in other stacks in that file: https://github.com/eclipse/che/blob/master/ide/che-core-ide-stacks/src/main/resources/stacks.json#L89